python/lsst/ts/electrometer/config_schema.py

```diff
@@ -75,6 +75,27 @@
     description: The timeout of the serial connection.
     type: integer
     default: 2
+  model_id:
```

Comment:
So there is a chance that we'll add a new Keysight electrometer in the future. So can you add a Brand as well?
So the header would be like:
BRAND: Keithley
MODEL: 6517b

Then you can add an example here in the docs:
`description: The model ID for the electrometer (e.g. 6517b).